src/documents/tests/test_document_model.py

```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 from unittest import mock
 
+import pytz
```

Comment:
Based on the [Django 4 release notes](https://docs.djangoproject.com/en/4.0/releases/4.0/) using `pytz` is now deprecated.  This should be simple enough to switch over to `zoneinfo`.  See the file `test_consumer.py` for how to support the multiple Python versions

Reply:
@stumpylog thanks for the hint. Done :)